code/modules/projectiles/projectile/magic.dm

```diff
@@ -161,7 +161,14 @@
 		M = shapeshift.stored
 		shapeshift.restore()
 
-	if(!istype(M) || M.stat == DEAD || M.notransform || (GODMODE & M.status_flags))
+	if(!istype(M) || istype(M, /mob/living/simple_animal/hostile/mimic/copy) || M.stat == DEAD || M.notransform || (GODMODE & M.status_flags))
+		return
+
+	if(ispAI(M))
+		var/mob/living/silicon/pai/Bot = M
+		var/holochassis = pick(Bot.possible_chassis)
+		Bot.set_holochassis(holochassis)
+		to_chat(Bot, span_boldnotice("Your holochassis form morphs into that of a [holochassis]."))
```

Comment:
```suggestion
		var/mob/living/silicon/pai/pai = M
		var/holochassis = pick(pai.possible_chassis)
		Bot.set_holochassis(holochassis)
		to_chat(pai, span_boldnotice("Your holochassis form morphs into that of a [holochassis]."))
```
Let's stay with snake_case here, not PascalCase.

Reply:
What code needs to be updated from PascalCase to snake_case?